### PR TITLE
KOGITO-3559 Expose Task States & Active Task States variables in Task…

### PR DESCRIPTION
**Pull request by bxf12315**
… Console Quarkus app

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [ ] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [ ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

File: task-console/src/main/resources/application.properties [hunk 1]
```diff
@@ -15,4 +15,7 @@ quarkus.http.auth.permission.roles1.policy=permit
 %keycloak.quarkus.oidc.logout.path=/logout
 %keycloak.quarkus.oidc.logout.post-logout-path=/
 %keycloak.quarkus.http.auth.permission.roles1.paths=/*
-%keycloak.quarkus.http.auth.permission.roles1.policy=authenticated
\ No newline at end of file
+%keycloak.quarkus.http.auth.permission.roles1.policy=authenticated
+
+kogtio.tasks.active.states.list=Ready-KOGITO-3559,Started-KOGITO-3559
```
Comment (pefernan) [inline task-console/src/main/resources/application.properties]: @bxf12315 I see this '-KOGITO-3559' here, I guess they shouldn't be right? The active states are Ready & Reserved

File: ui-packages/packages/task-console/webpack.common.js [hunk 2]
```diff
@@ -16,7 +16,9 @@ module.exports = {
     new webpack.EnvironmentPlugin({
       KOGITO_DATAINDEX_HTTP_URL: 'http://localhost:4000/graphql',
       KOGITO_APP_VERSION: 'DEV',
-      KOGITO_APP_NAME: 'Task Console'
+      KOGITO_APP_NAME: 'Task Console',
+      KOGITO_TASKS_STATES_LIST: 'Ready-KOGITO-3559,Started-KOGITO-3559,Completed,Aborted,Skipped',
+      KOGITO_TASKS_ACTIVE_STATES_LIST: 'Ready-KOGITO-3559,Started-KOGITO-3559'
```
Comment (pefernan) [inline ui-packages/packages/task-console/webpack.common.js]: Same here!

**Review by pefernan — CHANGES_REQUESTED** [review]

File: task-console/src/main/resources/application.properties (hunk 1 above)
Comment (pefernan) [reply task-console/src/main/resources/application.properties]: @bxf12315 typo on **kogtio**! I guess it should be **kogito**

**Review by AjayJagan — APPROVED** [review]

**Review by pefernan — APPROVED** [review]
+1 when green

**Comment by pefernan** [comment]
jenkins retest this please

**Comment by pefernan** [comment]
jenkins retest this please

**Comment by pefernan** [comment]
jenkins retest this please

**Comment by pefernan** [comment]
jenkins retest this please

**Comment by pefernan** [comment]
jenkins retest this

**Comment by bxf12315** [comment]
jenkins retest this

**Review by jstastny-cz — APPROVED** [review]

**Pull request merged**